proposals/lip-0042.md

```diff
@@ -108,7 +100,7 @@ getBlockReward(blockHeader):
   return defaultReward
 ```
 
-Here, `/` represents integer division, `isValidSeedReveal` is the function exposed by the [random module][research:random-module] and `impliesMaximalPrevotes` is the function exposed by the [BFT module][research:bft-module#specs].
+Here `isValidSeedReveal` is the function exposed by the [random module][research:random-module] and `impliesMaximalPrevotes` is the function exposed by the [BFT module][research:bft-module#specs].
```

Comment:
In other modules we used the notation `random.isValidSeedReveal` and `bft.impliesMaximalPrevotes`. It is also fine like this for me, but you may wanna consider this alternative.

Reply:
I will just leave it like this.